wifi_qrcode_generator/generator.py

```diff
@@ -0,0 +1,59 @@
+"""Generate WiFi QR code."""
+
+import qrcode
+
+
+AUTHENTICATION_TYPES = ['WPA', 'WEP', 'nopass']
+
+def wifi_code(ssid: str, hidden: str, authentication_type: str, password: str | None = None) -> str:
+    """Generate a WiFi code for the given parameters. 
+
+    The generated WiFi code can be rendered into a QR code to be scanned to join the network.
+
+    Args:
+        ssid: Network SSID.
+        hidden: Specify if the network is hidden.
+        authentication_type: Specify the authentication type. Supported types: 
+          `WPA`, `WEP`, `nopass`. 
+        password: Network password. If `authetication_type` is `None`, this argument should be set
+          to `None`.
+    Returns:
+        The WiFi code for the given parameters.
+    """
```

Comment:
You want a [`Raises:`](https://google.github.io/styleguide/pyguide.html#doc-function-raises) clause too (TypeError for password being wrong and ValueError for authentication_type being wrong)

Reply:
> You should not document exceptions that get raised if the API specified in the docstring is violated